scripts/pullDockerImages.sh

```diff
@@ -34,31 +34,31 @@ dockerTag() {
 
   for IMAGE in $IMAGELIST; do
     echo
-    echo "Image: $IMAGE"
-    if ! docker pull $NEXUS_URL/$ORG_NAME-$IMAGE:$LATEST_TAG > /dev/null; then
+    echo "Image: fabric-$IMAGE"
+    if ! docker pull $ARTIFACTORY_URL/fabric-$IMAGE:$LATEST_TAG > /dev/null; then
            echo  "FAILED: Docker Pull Failed on $IMAGE"
            exit 1
     fi
-    docker tag $NEXUS_URL/$ORG_NAME-$IMAGE:$LATEST_TAG $ORG_NAME-$IMAGE
-    docker tag $NEXUS_URL/$ORG_NAME-$IMAGE:$LATEST_TAG $ORG_NAME-$IMAGE:$LATEST_TAG
-    docker tag $NEXUS_URL/$ORG_NAME-$IMAGE:$LATEST_TAG $ORG_NAME-$IMAGE:$VERSION
+    docker tag $ARTIFACTORY_URL/fabric-$IMAGE:$LATEST_TAG $ORG_NAME/fabric-$IMAGE
+    docker tag $ARTIFACTORY_URL/fabric-$IMAGE:$LATEST_TAG $ORG_NAME/fabric-$IMAGE:$LATEST_TAG
+    docker tag $ARTIFACTORY_URL/fabric-$IMAGE:$LATEST_TAG $ORG_NAME/fabric-$IMAGE:$VERSION
     if [ $IMAGE == javaenv ]; then
-        docker tag $NEXUS_URL/$ORG_NAME-$IMAGE:$LATEST_TAG $ORG_NAME-$IMAGE:$ARCH-$VERSION
+        docker tag $ARTIFACTORY_URL/fabric-$IMAGE:$LATEST_TAG $ORG_NAME/fabric-$IMAGE:$ARCH-$VERSION
     fi
-    echo "$ORG_NAME-$IMAGE:$LATEST_TAG"
-    echo "Deleting Nexus docker images: $IMAGE"
-    docker rmi -f $NEXUS_URL/$ORG_NAME-$IMAGE:$LATEST_TAG
+    echo "Pulled: $ORG_NAME/fabric-$IMAGE:$LATEST_TAG"
+    echo "Deleting Artifactory docker images: fabric-$IMAGE"
+    docker rmi -f $ARTIFACTORY_URL/fabric-$IMAGE:$LATEST_TAG
   done
 }
 
 dockerThirdParty() {
   for IMAGE in kafka zookeeper couchdb; do
-    echo "$ORG_NAME-$IMAGE"
-    if ! docker pull $ORG_NAME-$IMAGE:latest > /dev/null; then
+    echo "$ORG_NAME/fabric-$IMAGE"
+    if ! docker pull $ORG_NAME/fabric-$IMAGE:latest > /dev/null; then
        echo  "FAILED: Docker Pull Failed on $IMAGE"
        exit 1
     fi
-    docker tag $ORG_NAME-$IMAGE:latest $ORG_NAME-$IMAGE:$ARCH-latest
+    docker tag $ORG_NAME/fabric-$IMAGE:latest $ORG_NAME/fabric-$IMAGE:$ARCH-latest
   done
 }
```

Comment:
In fabric master branch, we are pulling these images from it's source repos. See here https://github.com/hyperledger/fabric/blob/master/Makefile#L153-L157. Update the same here.

Reply:
Created JIRA for this task FAB-17295. Will address this comment in next patch.